Tests/Controllers/PeopleController.cs

```diff
@@ -50,6 +51,13 @@ public Person PostPerson(string id, Person person)
         {
             person.Identifier = id;
             return person;
+        }
+
+        [HttpPost]
+        [Route("relationships/people")]
+        public JObject PostParams([FromBody] JObject bodyParams)
```

Comment:
bodyParams should include a flattened version of the attributes and relationships, yeah?

currently, it only gives me attributes